library/src/amd_detail/rocblaslt/include/rocblaslt-types.h

```diff
@@ -167,7 +167,9 @@ typedef enum rocblaslt_epilogue_
     ROCBLASLT_EPILOGUE_DGELU         = 192,
     ROCBLASLT_EPILOGUE_DGELU_BGRAD   = 208,
     ROCBLASLT_EPILOGUE_BGRADA        = 256,
-    ROCBLASLT_EPILOGUE_BGRADB        = 512
+    ROCBLASLT_EPILOGUE_BGRADB        = 512,
+    ROCBLASLT_EPILOGUE_SWISH         = 1024,
```

Comment:
Please shift the enum number to a larger one in case that has confliction with cublaslt enum in the future.
Besides, we use _EXT postfix for hipblaslt specific extension.
ex: ROCBLASLT_EPILOGUE_SWISH_EXT   = 2^16
ROCBLASLT_EPILOGUE_SWISH_BIAS_EXT   = 2^16 + 4